MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameAndPasswordEndToEndTests.swift

```diff
@@ -231,56 +268,28 @@ final class MSALNativeAuthSignUpUsernameAndPasswordEndToEndTests: MSALNativeAuth
         signUpAttributesRequiredDelegate.signInAfterSignUpState?.signIn(delegate: signInAfterSignUpDelegate)
 
         await fulfillment(of: [signInExp])
-        checkSignInAfterSignUpDelegate(signInAfterSignUpDelegate)
-    }
-    
-    // Hero Scenario 2.2.2. Sign in – Email and Password on MULTIPLE screens (Email & Password)
-    func test_signInAndSendingCorrectPasswordResultsInSuccess() async throws {
-        throw XCTSkip("Skipping this test because native auth KeyVault is missing")
-        guard let sut = initialisePublicClientApplication(clientIdType: .code) else {
-            XCTFail("Missing information")
-            return
-        }
-        
-        let signInExpectation = expectation(description: "signing in")
-        let passwordRequiredExpectation = expectation(description: "verifying password")
-        let signInDelegateSpy = SignInStartDelegateSpy(expectation: signInExpectation)
-        let signInPasswordRequiredDelegateSpy = SignInPasswordRequiredDelegateSpy(expectation: passwordRequiredExpectation)
-
-        let username = ProcessInfo.processInfo.environment["existingPasswordUserEmail"] ?? "<existingPasswordUserEmail not set>"
-        let password = ProcessInfo.processInfo.environment["existingUserPassword"] ?? "<existingUserPassword not set>"
-
-        sut.signIn(username: username, correlationId: correlationId, delegate: signInDelegateSpy)
-
-        await fulfillment(of: [signInExpectation])
-
-        XCTAssertTrue(signInDelegateSpy.onSignInPasswordRequiredCalled)
-        XCTAssertNotNil(signInDelegateSpy.newStatePasswordRequired)
-
-        // Now submit the password..
-
-        signInDelegateSpy.newStatePasswordRequired?.submitPassword(password: password, delegate: signInPasswordRequiredDelegateSpy)
-
-        await fulfillment(of: [passwordRequiredExpectation])
-
-        XCTAssertTrue(signInPasswordRequiredDelegateSpy.onSignInCompletedCalled)
-        XCTAssertNotNil(signInPasswordRequiredDelegateSpy.result?.idToken)
-        XCTAssertEqual(signInPasswordRequiredDelegateSpy.result?.account.username, username)
+        checkSignInAfterSignUpDelegate(signInAfterSignUpDelegate, expectedUsername: username)
     }
 
-    // Hero Scenario 2.1.5. Sign up - with Email verification as FIRST step & Custom Attributes over MULTIPLE screens (Email & Password)
+    // Hero Scenario xxx. Sign up - with Email verification as FIRST step & Custom Attributes over MULTIPLE screens (Email & Password)
```

Comment:
Do we have a hero scenario for this?
If not, we can just omit the first sentence like this:
```suggestion
    // Sign up - with Email verification as FIRST step & Custom Attributes over MULTIPLE screens (Email & Password)
```